src/_includes/web-setup-deprecate.md

```diff
@@ -1,2 +1,2 @@
 {:.bs-callout-warning}
-The Web Setup Wizard is being deprecated in Magento 2.3.6 and will be removed in Magento 2.4.0. After it is removed, you must use the command line to [install](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli.html) or [upgrade](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html) Magento, as well as [extensions](https://devdocs.magento.com/extensions/install/).
\ No newline at end of file
+The Web Setup Wizard is being deprecated in Magento 2.3.6 and removed in Magento 2.4.0. You must use the command line to [install](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli.html) or [upgrade](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html) Magento, as well as [extensions](https://devdocs.magento.com/extensions/install/).
```

Comment:
Since this is the 2.3 guide, it is technically not removed from this version. So a slight tweak here for clarity.

```suggestion
The Web Setup Wizard is deprecated as of Magento 2.3.6 and removed in Magento 2.4.0. As an alternative, use the command line to [install](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli.html) or [upgrade](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html) Magento, as well as [extensions](https://devdocs.magento.com/extensions/install/).
```